ffi/src/lib.rs

```diff
@@ -1755,3 +1776,275 @@ pub(crate) unsafe fn get_pipeline_options(
         PipelineRetryStrategy::new(info.retry_server_error, info.retry_connection_error),
     )
 }
+
+/// Creates an OpenTelemetry span with the given name and returns a pointer to the span as u64.
+///
+/// # Safety
+/// * `request_type` must be a valid request type.
+#[unsafe(no_mangle)]
+pub unsafe extern "C" fn create_otel_span(request_type: RequestType) -> u64 {
```

Comment:
```suggestion
pub extern "C-unwind" fn create_otel_span(request_type: RequestType) -> u64 {
```
Isn't unsafe, because don't have `unsafe` blocks, but it has a panic instead. Should it return 0 instead or any smarter structure (like command response which has a value or error)?

Reply:
Talk with @jonathanl-bq about "C" vs "C-unwind"